packages/map/src/map/index.jsx

```diff
@@ -66,12 +76,14 @@ Map.defaultProps = {
   lockedMobile: true,
   noAttribution: false,
   fitPadding: 20,
+  translations: {},
 };
 
 Map.propTypes = {
   className: PropTypes.string,
   children: PropTypes.node,
 
+  translations: PropTypes.object.isRequired,
```

Comment:
we could describe the shape of the object here since we already know it

Reply:
would work now. But it can grow. Don't want to write every key in propType check. We don't do it in calendar and cookie banner